build.gradle

```diff
@@ -5,8 +5,8 @@ plugins {
 }
 
 group 'house.inksoftware'
-version '29.0.0'
-sourceCompatibility = '11'
+version '29.1.0'
```

Comment:
I wonder if that would affect other components that use java 8 / 12? Wouldn't we face a problem if we would bump them to this version?

Reply:
Agreed, I have bumped to 30.0.0. 29.**x.x** will be support for 29